build_scripts/build_usd.py

```diff
@@ -682,6 +728,14 @@ def AnyPythonDependencies(deps):
 
 def InstallZlib(context, force, buildArgs):
     with CurrentWorkingDirectory(DownloadURL(ZLIB_URL, context, force)):
+        if context.targetIos:
```

Comment:
is it possible to suppress the creation of the zlib tests instead? Also, is zlib part of the iOS SDK? If it is maybe we shouldn't be downloading and installing it at all?

Reply:
After looking into it, I think Zlib's cmakelist is not very friendly for turning off tests.
Using -DBUILD_TESTING=OFF did not disable the test's generation as items.
The cmakelist simply always adds the executables.
<img width="735" alt="image" src="https://user-images.githubusercontent.com/9114265/218801363-7f8fc1f1-0f6b-466a-8659-557cb9f42a27.png">
From the zlib CmakeList

That then leads to an error in the build automation for Xcode projects that need bundle identifiers (iOS ones do)
<img width="1612" alt="image" src="https://user-images.githubusercontent.com/9114265/218801566-18a03514-bc29-4a88-b479-a52eb45219c1.png">

Further down the line we could create automations that edit Xcode files, but adding an arbitrary bundle identifier meaningful for the building user is also something that is not very intuitive to automate as bundle identifiers should be unique for each user and probably we should not create identifiers for users in general.

Reply:
If we must patch, then lets suppress them, rather than fix them, please.

Reply:
Cool! Have modified it that way - the modifications I'm making, I might add as a final commit but later on possibly move into the nice commit chain